planet.spec.rb

```diff
@@ -0,0 +1,20 @@
+# require 'minitest/autorun'
```

Comment:
If you wrote tests for `Planet`, you'd probably want to make the name of the file `planet_spec.rb` and not `planet.spec.rb`